go/cmd/gh-ost/main.go

```diff
@@ -139,6 +139,7 @@ func main() {
 	criticalLoad := flag.String("critical-load", "", "Comma delimited status-name=threshold, same format as --max-load. When status exceeds threshold, app panics and quits")
 	flag.Int64Var(&migrationContext.CriticalLoadIntervalMilliseconds, "critical-load-interval-millis", 0, "When 0, migration immediately bails out upon meeting critical-load. When non-zero, a second check is done after given interval, and migration only bails out if 2nd check still meets critical load")
 	flag.Int64Var(&migrationContext.CriticalLoadHibernateSeconds, "critical-load-hibernate-seconds", 0, "When non-zero, critical-load does not panic and bail out; instead, gh-ost goes into hibernation for the specified duration. It will not read/write anything from/to any server")
+	flag.StringVar(&migrationContext.OptimizerSwitch, "optimizer-switch", "", "Optimizer switch params, eg: prefer_ordering_index=on")
```

Comment:
```suggestion
	flag.StringVar(&migrationContext.OptimizerSwitch, "optimizer-switch", "prefer_ordering_index=on", "optimizer_switch param")
```
I think we can make `prefer_ordering_index=on` a default setting as it is already the MySQL default and could make results more correct

For example, the min/max value queries for the migration rely on the index being ordered. "prefer" is good, but this might highlight a need for a `FORCE INDEX` when we get the min/max values longer term 🤔 

Any thoughts here @dm-2 / @rashiq / @morgo?

Reply:
`Prefer_ordering_index` is `ON` by default, but my opinion is that this is only for backward compatibility. @cyrinux are you sure you don't need to turn it `OFF` for your use-case?

I don't like `Prefer_ordering_index` as an optimization, since my experience has been it often makes mistakes (leading to very bad performance). We are talking about the same issue [written about here](https://blog.jcole.us/2019/09/30/reconsidering-access-paths-for-index-ordering-a-dangerous-optimization-and-a-fix/).

We have ran into this problem on my team (not for gh-ost, for other workloads), but our workaround has been to use `FORCE INDEX`. This is only because the switch shipped after 5.7-GA so it's not in Aurora v2.

Reply:
@morgo I'm sure we toggle `Prefer_ordering_index=on` 😉 
And yes, its the default behavior but not for me. @morgo @timvaillancourt should I so add FORCE INDEX to the `SELECT COUNT(*)` as this make sense to always use it for this kind of query ?